### PR TITLE
fix: upgrade to latest slimerjs version

### PR DESCRIPTION
**Pull request by mattlewis92**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   ],
   "author": "Nicolas Froidure <nfroidure@elitwork.com>",
   "dependencies": {
-    "slimerjs": "0.9.6-2"
+    "slimerjs": "~0.906.1"
```
Comment (dignifiedquire) [inline package.json]: well that's an interesting version number :D

**Comment by dignifiedquire** [comment]
Thanks :octocat:

File: package.json (hunk 1 above)
Comment (mattlewis92) [inline package.json]: I've seen stranger before 😄  https://github.com/dtinth/babel-plugin-__coverage__/tags

**Comment by mattlewis92** [comment]
@dignifiedquire any chance you could cut a new release with this patch? 😄 

Also this can be closed now: https://github.com/karma-runner/karma-slimerjs-launcher/pull/15

**Comment by dignifiedquire** [comment]
Sure thing, new version is released

**Comment by mattlewis92** [comment]
Thank you! 👍

File: package.json (hunk 1 above)
Comment (graingert) [inline package.json:20]: It's because -x versions aren't supported properly in npm

**Pull request merged**